apps/arm/include/ClawConfig.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "ActuatorControllerManager.h"
+#include "Current.h"
+#include "DCMotor.h"
+#include "LookupTable.h"
+#include "OpenLoop.h"
+#include "PID.h"
+#include "Pololu37D.h"
+#include "Position.h"
+#include "Velocity.h"
+#include "hw_bridge.h"
+
+namespace Claw {
+
+namespace Internal {
+
+static Encoder::Pololu37D encoder({ENC_QUAD_CLAW_A, ENC_QUAD_CLAW_B, NC, 0, GPIO::QEI::Encoding::X2_ENCODING});
+
+static Actuator::DCMotor motor(MTR_PWM_TRNTBL, MTR_DIR_TRNTBL, false);
+
+static Sensor::CurrentSensor currentSensor;
+
+static PID::PID velPID({1, 0, 0, -1, 1, 0, false, false});
```

Comment:
I think for readability it would be helpful to have  commented of what the parameters are representing

Reply:
the PID class details all of them at the top. is that fine?

Reply:
this would get kinda overwhelming imo. bc we have pid objects in many header files and the explanation won't be as good as in the PID header. i think its reasonable that if someone wants to check what the constructor args are for to check the constructor.

Reply:
I'd say if we have so many constructor variables, we should just be passing a reference to a config struct. More readable since in the struct, you can initialize members by their names

Reply:
then you have like 4 config structs per header file lol

Reply:
just for PID that is. That's what I had at the start and decided it looked too cluttered and reduced readability